internal/reminder/check_last_pin.go

```diff
@@ -21,8 +21,6 @@ func hasLastPin(ctx context.Context, client bot.Client, logger log.Logger, incid
 			ctx,
 			log.Trace(),
 			log.Reason("LastPin"),
-			log.NewValue("channelID", incident.ChannelId),
```

Comment:
Suggestion: rename the parameter logger to logWriter to make it clear that it already contains the log fields.